components/expenses/MarkExpenseAsUnpaidBtn.js

```diff
@@ -47,7 +47,7 @@ const MarkExpenseAsUnpaidBtn = ({ id, markExpenseAsUnpaid }) => {
         <StyledButton
           onClick={() => setState({ ...state, showProcessorFeeConfirmation: true })}
           mt={2}
-          buttonStyle="secondary"
+          buttonStyle="primary"
```

Comment:
In my view this shouldn't be `primary` nor `secondary` because this is not a call to action that we want to promote in any way, so standard style should apply.

```suggestion
```